contracts/contracts/coordination/Coordinator.sol

```diff
@@ -184,11 +198,15 @@ contract Coordinator is AccessControlDefaultAdminRules {
         return ritual.participant;
     }
 
-    function initiateRitual(
+    function _initiateRitual(
```

Comment:
Should we walk back this change i.e. change `_initiateRitual` -> `initiatRitual` and remove the function on L254, since the params are now the same?

Reply:
✔️